src/Support/Types.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Spatie\Dns\Support;
+
+class Types
+{
+    public const TYPES = [
+        DNS_A => 'A',
+        DNS_AAAA => 'AAAA',
+        DNS_CNAME => 'CNAME',
+        DNS_NS => 'NS',
+        DNS_SOA => 'SOA',
+        DNS_MX => 'MX',
+        DNS_SRV => 'SRV',
+        DNS_TXT => 'TXT',
+        DNS_CAA => 'CAA',
+        // DNS_NAPTR => 'NAPTR',
+    ];
+
+    public function toNames(int $flags): array
+    {
+        $types = [];
+
+        foreach (self::TYPES as $flag => $type) {
+            if ($flags & $flag) {
+                $types[$flag] = $type;
+            }
+        }
+
+        return $types;
+    }
+
+    public function toFlags(array $types): int
```

Comment:
I'm not sure to what this function does, could you make this one a bit more readable too?

Reply:
This method is the inverse of the `toNames()` so you put in a list of strings and get out a bit-map.

Which line(s) exactly should I make clearer? And is it about variable naming? Or the bit-operator? Or should I use the single assignment operator `|=` instead of `$flags = $flags | $flag;`?